From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeff Genovy <29107334+jefgen@users.noreply.github.com>
Date: Thu, 10 Dec 2020 13:31:59 -0800
Subject: MSFT-PATCH: Add closing comments to ifdefs for Windows SDK header
 tools

The Windows SDK header tools need to have the closing #endif contain a
comment for the opening #if so that it can match them up.

diff --git a/icu/icu4c/source/common/unicode/uloc.h b/icu/icu4c/source/common/unicode/uloc.h
index 3addb847e7fe6f822ac325314ae2214d18af035e..2c33c02bfd022ee9f25eecff5af1e36a430c41ef 100644
--- a/icu/icu4c/source/common/unicode/uloc.h
+++ b/icu/icu4c/source/common/unicode/uloc.h
@@ -841,7 +841,7 @@ typedef enum ULocAvailableType {
    * @internal
    */
   ULOC_AVAILABLE_COUNT
-#endif
+#endif // U_HIDE_INTERNAL_API
 } ULocAvailableType;
 
 /**
diff --git a/icu/icu4c/source/i18n/unicode/unumberrangeformatter.h b/icu/icu4c/source/i18n/unicode/unumberrangeformatter.h
index e28fe51aa6ae34331c8be12d7d45e9c107a767e8..a90861d9dd0c09fa73d539820eb4c24773bc2bb3 100644
--- a/icu/icu4c/source/i18n/unicode/unumberrangeformatter.h
+++ b/icu/icu4c/source/i18n/unicode/unumberrangeformatter.h
@@ -193,7 +193,7 @@ typedef enum UNumberRangeIdentityResult {
      * @internal
      */
     UNUM_IDENTITY_RESULT_COUNT
-#endif
+#endif // U_HIDE_INTERNAL_API
 
 } UNumberRangeIdentityResult;
 
